app/index.tsx

```diff
@@ -4,6 +4,9 @@ import { useRouter } from 'expo-router';
 import { ArrowRight, Mic, Book, BarChart, Users, Play } from 'lucide-react-native';
 import { LinearGradient } from 'expo-linear-gradient';
 
+import PricingCard from '@/components/PricingCard';
```

Comment:
Do you think we could store these in components/landing_page as it's likely we'll only ever use them there?

Reply:
Yes that sounds better